src/generate/ast/node.rs

```diff
@@ -89,3 +82,87 @@ pub enum Core {
     With { resource: Box<Core>, expr: Box<Core> },
     WithAs { resource: Box<Core>, alias: Box<Core>, expr: Box<Core> },
 }
+
+#[derive(Debug, PartialEq, Eq, Hash, Clone)]
+pub enum CoreOp {
```

Comment:
This enum should be split in two:
- Operation which designate an operation within python (`Ge`, `Le`, `Pow`, etc.).
- Operations which designate names of functions which overload operations (AddAssign, SubAssign ,etc.).